db.json

```
@@ -21,6 +21,39 @@
       "cost": 0,
       "services": [],
       "id": 2
+    },
+    {
+      "name": "Projeto Design Bracell",
+      "budget": "10.000",
+      "category": {
+        "id": "3",
+        "name": "Design"
+      },
+      "cost": 0,
+      "services": [],
+      "id": 3
+    },
+    {
+      "category": {
+        "id": "4",
+        "name": "Planejamento"
+      },
+      "name": "Projeto Planejamento",
+      "budget": "16.000",
```

Validar somente com pontos
```suggestion
      "budget": "16.000",
```